docs/resources/projects/libraries.md

```diff
@@ -226,6 +226,89 @@ You can easily upgrade to newer versions of the libraries as they become availab
 
 ![update-library](imgs/update-library.avif)
 
+## Library Values
+
+**Library values** are essentially variables created and used by a library publisher and intended to have their values set by the library consumer. These values allow library publishers to create configurable variables that are useful in different contexts, such as API keys, global settings, or other project-specific configurations. These values allow consumers to input specific data required for the library to function properly in their project.
```

Comment:
maybe we say "library author" and "library user" instead of library publisher and library consumer?

Reply:
Yes, updated now.